components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -622,6 +633,38 @@ protected boolean handlePostSetUserClaimValues(Event event, String userName, Use
                 auditAccountLock(AuditConstants.ACCOUNT_LOCKED, userName, userStoreDomainName, isAdminInitiated,
                         null, AuditConstants.AUDIT_SUCCESS, true);
             } else if (lockedStates.LOCKED_UNMODIFIED.toString().equals(lockedState.get())) {
+                boolean notificationOnLockIncrement = false;
+                try {
+                    notificationOnLockIncrement = Boolean.parseBoolean(AccountUtil.getConnectorConfig(AccountConstants
+                            .NOTIFY_ON_LOCK_DURATION_INCREMENT, tenantDomain));
+                } catch (IdentityEventException e) {
+                    log.warn("Error while reading notification on lock increment property in account lock handler");
```

Comment:
Add some context information to the warning log.

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-handler-account-lock/pull/95/commits/fa488a13c0704630784fb41ac637d26ecf72ebe7